srcs/exec/redirects.c

```diff
@@ -77,12 +77,8 @@ static t_result	expand_and_exec_redirect_all(t_ast *self_node, \
 	return (SUCCESS);
 }
 
-t_result	close_exit_fd(t_ast *self_node)
+t_result	close_prod_fd_id_exit(t_ast *self_node)
```

Comment:
口では if って言ってたような気がします…！(107 行目も)
```suggestion
t_result	close_prod_fd_if_exit(t_ast *self_node)
```

Reply:
idとは...😇